src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -13,44 +13,76 @@
-void func_80A7506C(EnIk* this);
-void func_80A7510C(EnIk* this, GlobalContext* globalCtx);
-void func_80A751C8(EnIk* this);
-void func_80A75260(EnIk* this, GlobalContext* globalCtx);
-void func_80A753D0(EnIk* this);
-void func_80A7545C(EnIk* this, GlobalContext* globalCtx);
-void func_80A754A0(EnIk* this);
-void func_80A75530(EnIk* this, GlobalContext* globalCtx);
-void func_80A755F0(EnIk* this);
-void func_80A7567C(EnIk* this, GlobalContext* globalCtx);
-void func_80A758B0(EnIk* this, GlobalContext* globalCtx);
-void func_80A75A38(EnIk* this, GlobalContext* globalCtx);
-void func_80A75FA0(Actor* thisx, GlobalContext* globalCtx);
-void func_80A76798(Actor* thisx, GlobalContext* globalCtx);
-void func_80A7748C(EnIk* this, GlobalContext* globalCtx);
-void func_80A774BC(EnIk* this, GlobalContext* globalCtx);
-void func_80A774F8(EnIk* this, GlobalContext* globalCtx);
-void func_80A77844(EnIk* this, GlobalContext* globalCtx);
-void func_80A779DC(EnIk* this, GlobalContext* globalCtx);
-void func_80A77AEC(EnIk* this, GlobalContext* globalCtx);
-void func_80A77B0C(EnIk* this, GlobalContext* globalCtx);
-void func_80A77B3C(EnIk* this, GlobalContext* globalCtx);
-void func_80A77ED0(EnIk* this, GlobalContext* globalCtx);
-void func_80A77EDC(EnIk* this, GlobalContext* globalCtx);
-void func_80A78160(EnIk* this, GlobalContext* globalCtx);
-void func_80A781CC(Actor* thisx, GlobalContext* globalCtx);
+void EnIk_SetupStandUp(EnIk* this);
+void EnIk_StandUp(EnIk* this, GlobalContext* globalCtx);
+void EnIk_Idle(EnIk* this, GlobalContext* globalCtx);
+void EnIk_SetupWalkOrRun(EnIk* this);
+void EnIk_WalkOrRun(EnIk* this, GlobalContext* globalCtx);
+void EnIk_SetupVerticleAttack(EnIk* this);
+void EnIk_VerticleAttack(EnIk* this, GlobalContext* globalCtx);
+void EnIk_SetupPullOutAxe(EnIk* this);
+void EnIk_PulloutAxe(EnIk* this, GlobalContext* globalCtx);
+void EnIk_SetupDoubleHorizontalAttack(EnIk* this);
+void EnIk_DoubleHorizontalAttack(EnIk* this, GlobalContext* globalCtx);
+void EnIk_SetupRecoverFromHorizontalAttack(EnIk* this);
+void EnIk_RecoverFromHorizontalAttack(EnIk* this, GlobalContext* globalCtx);
+void EnIk_SetupSingleHorizontalAttack(EnIk* this);
+void EnIk_SingleHorizontalAttack(EnIk* this, GlobalContext* globalCtx);
+void EnIk_SetupStopAndBlock(EnIk* this);
+void EnIk_StopAndBlock(EnIk* this, GlobalContext* globalCtx);
+void EnIk_ReactToAttack(EnIk* this, GlobalContext* globalCtx);
+void EnIk_Death(EnIk* this, GlobalContext* globalCtx);
+void EnIk_AxeAttackHitPlayer(Actor* thisx, GlobalContext* globalCtx);
+void EnIk_DrawParamType(Actor* thisx, GlobalContext* globalCtx);
+void EnIk_UpdateCs_03(EnIk* this, GlobalContext* globalCtx);
+void EnIk_UpdateCs_04(EnIk* this, GlobalContext* globalCtx);
+void EnIk_UpdateCs_05(EnIk* this, GlobalContext* globalCtx);
+void EnIk_CsNabooruDefeatDraw(EnIk* this, GlobalContext* globalCtx);
+void EnIk_HandleSubscenesByNpcAction(EnIk* this, GlobalContext* globalCtx);
```

Comment:
These all deal with the Nabooru fight, so there are two different cutscenes, but they handle different subscenes within the two cutscenes.

Reply:
Err, is "subscene" to be understood as "part of scene" or "part of cutscene"
Since everything happens in the same scene I don't understand why "part of scene" would be used here?

Also this function is used by all cutscene actions so it's not specific to a specific part of the cutscenes involving this actor

Reply:
bump

Reply:
When I say subscene I am referring to different defining actions that happen at the beginning of a new subscene is a change of camera and the actor performing a new action.